test/test_yaml.py

```diff
@@ -189,3 +191,14 @@ def test_update_fn(self):
         data = to_yaml(data)
         card.update(data)
         self.assertEqual(card.formatted_name, fn)
+
+    def test_parse_field_with_colon(self):
+        card = create_test_card()
+        fn = 'me myself and i'
```

Comment:
this test has nothing to do with `FN` so please remove that. Tests should be as short and simple as possible.

Reply:
Will do. I think the FN needs to be set or `card.update()` will fail, but I can inline it.